services/userServices.js

```diff
@@ -6,7 +6,7 @@ const addUserService = async (user) => {
   return resData;
 };
 
-const isAllValid = (data) => {
+const isInValid = (data) => {
```

Comment:
I think you can replace like
` if (
    data.name &&
    data.mobile_no &&
    data.email &&
    data.password &&
    data.emp_code
  ) {
    return false;
  }
  return true`
with 
` return (data.name &&
    data.mobile_no &&
    data.email &&
    data.password &&
    data.emp_code)`

Reply:
yah, the better way!